internal/cmd/egctl/testdata/translate/out/invalid-securitypolicy.all.yaml

```diff
@@ -108,8 +108,8 @@ securityPolicies:
         namespace: envoy-gateway-system
       conditions:
       - lastTransitionTime: null
-        message: 'Invalid SecurityPolicy: no security policy is specified.'
```

Comment:
does this test case need to be revisited ? its called `invalid-securitypolicy` but `Accepted=true`

Reply:
Good catch.

Reply:
Dleted this test file.